src/index.test.js

```diff
@@ -85,6 +88,36 @@ test('options: logLevel sets the log level', () => {
   })
 })
 
+test('options: scripts logs command text when false', () => {
+  const {runPackageScript, infoSpy} = setup()
+  const scriptConfig = {test: {script: 'echo test'}}
+  const options = {scripts: true}
+  return runPackageScript({
+    scriptConfig,
+    scripts: ['test'],
+    options,
+  }).then(() => {
+    expect(infoSpy.firstCall.args[0]).toMatch(
```

Comment:
Why don't we do `toMatchSnapshot` instead? Will be easier to update in the future.